devtools/conda-envs/base.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - sqlalchemy>=1.3
   - psycopg2
   - postgresql
+  - alembic
```

Comment:
This needs to be added to the `generate_envs.py` in this folder and then run so that this change propagates to all envs.